### PR TITLE
Brushup Barbarian Rage automation

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

**Review by SpartanCPA — DISMISSED** [review]

File: packs/data/classfeatures.db/animal-instinct.json [hunk 1]
```diff
@@ -23,51 +23,393 @@
...
-                        "value": "Compendium.pf2e.classfeatures.Wolf Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Wolf",
+                        "value": "wolf"
                     }
                 ],
-                "flag": "animal",
                 "key": "ChoiceSet",
-                "prompt": "PF2E.SpecificRule.Barbarian.Animal.Prompt"
+                "prompt": "PF2E.SpecificRule.Barbarian.Animal.Prompt",
+                "rollOption": "feature:instinct:animal"
```
Comment (stwlam) [inline packs/data/classfeatures.db/animal-instinct.json:66]: Can we switch this to `animal-instinct`? That makes it look like a class feature item, and I'd rather not risk that biting us in the future.

**Comment by stwlam** [comment]
> Still need to be able to add reach to strikes

You can add reach to strikes. What's the use case?

File: packs/data/classfeatures.db/animal-instinct.json (hunk 1 above)
Comment (TikaelSol) [reply packs/data/classfeatures.db/animal-instinct.json]: Done

**Comment by TikaelSol** [comment]
Animal instinct's specialization ability `The frog's tongue attack and deer's antler attack gain reach 10 feet`

File: packs/data/classfeatures.db/animal-instinct.json [hunk 2]
```diff
@@ -23,51 +23,393 @@
...
+                    "class:barbarian",
+                    {
+                        "gte": [
+                            "self:level",
+                            7
+                        ]
+                    }
+                ],
+                "property": "weapon-traits",
+                "value": "reach-10"
```
Comment (stwlam) [inline packs/data/classfeatures.db/animal-instinct.json:412]:
```suggestion
                "value": "reach"
```

File: static/lang/re-en.json [hunk 3]
```diff
@@ -2254,4 +2269,4 @@
             }
         }
     }
-}
+}
```
Comment (stwlam) [inline static/lang/re-en.json:2272]:
```suggestion
}

```

**Pull request merged**